examples/speaker_tasks/diarization/README.md

```diff
@@ -31,12 +31,13 @@ Recommended models on NGC:
 ## Performance
 Diarization Error Rate (DER) table of `titanet_large.nemo` model on well known evaluation datasets. 
 
-|         Evaluation<br>Condition     | NIST SRE 2000 | AMI<br>(Lapel) | AMI<br>(MixHeadset) | CH109 |
-|:-----------------------------------:|:-------------:|:--------------:|:-------------------:|:-----:|
-|  Oracle VAD <br>KNOWN # of Speakers  |      6.73     |      2.03      |         1.73        |  1.19 |
-| Oracle VAD<br> UNKNOWN # of Speakers |     5.38     |      2.03      |         1.89        |  1.63 |
+|         Evaluation Condition           |   AMI(Lapel)   | AMI(MixHeadset)     |   CH109  | NIST SRE 2000 |  
+|:--------------------------------------:|:--------------:|:-------------------:|:--------:|:-------------:|
+|       Domain Configuration             |   Meeting      |        Meeting      |Telephonic| Telephonic    |
+|  Oracle VAD <br> KNOWN # of Speakers   |     1.28       |         1.07        |  0.56    |     5.62      |
+|  Oracle VAD <br> UNKNOWN # of Speakers |     1.28       |         1.4         |  0.88    |     4.33      | 
 
-* All models were tested using embedding extractor with window size 1.5s and shift length 0.75s
+* All models were tested using the domain specific `.yaml` files which can be found in `conf/inference/` folder.
```

Comment:
update line numbers 50, 61, 111, and 120.

Reply:
144 and 154

Reply:
269

Reply:
All above are related to updating offline_diarization.py to offline_diar_infer.py and similarly *_with_asr*_.py filenames

Reply:
all replaced